elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+respawn limit 10 30
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
+env HOME=/home/{{ pillar.elife.deploy_user.username }}
```

Comment:
This might be the reason why its not running, is there a log file that will show errors with upstart that I can inspect?